### PR TITLE
Fixes issues in webpack config example

### PR DESCRIPTION
**Pull request by alanbuchanan**
Fixes issues in webpack config example

Fix grammar, implement build folder, update references and trees

Make reference to gitignore boilerplate site not file

Fix help bullet points

Add a quick instructions summary section

Fix entry point using path.join

Appropriately label js code block

### CODE REVIEW

File: Setting-Up-A-React-ES6-Webpack-Project.md [hunk 1]
```diff
@@ -235,14 +235,13 @@ Here is what the `webpack.config.js` file looks like:
 
 ```js
 var path = require('path');
-var context = path.join(__dirname, 'src')
```
Comment (abhisekp) [inline Setting-Up-A-React-ES6-Webpack-Project.md]:
```js
var buildPath = path.join(__dirname, 'dist');
var srcPath = path.join(__dirname, 'src');
```

File: Setting-Up-A-React-ES6-Webpack-Project.md [hunk 2]
```diff
@@ -235,14 +235,13 @@ Here is what the `webpack.config.js` file looks like:
...
 
 module.exports = {
-  context: context,
+  context: path.join(__dirname, "src"),
   entry: "./js/client.js",
   output: {
-      path: context,
-      filename: "./js/client.min.js"
+      path: __dirname + "/src/",
```
Comment (abhisekp) [inline Setting-Up-A-React-ES6-Webpack-Project.md]: use `path.join`

**Comment by alanbuchanan** [comment]
@abhisekp please may you take a look over the most recent commit in case I have written something inaccurate

**Comment by abhisekp** [comment]
@Rafase282 merge and release the Kraken :+1:  :shipit:

File: Setting-Up-A-React-ES6-Webpack-Project.md [hunk 3]
```diff
@@ -1,39 +1,45 @@
...
 ## Initializing the Project
 
 The first thing to do is open our terminal/command line. We need to install Webpack and Webpack Dev Server globally.
 
-[Help: More about installing Node Modules globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally)
+- [Help: More about installing Node Modules globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally)
 
 ```
-npm install webpack webpack-dev-server –g
+npm install webpack –g && npm install webpack-dev-server -g
```
Comment (abhisekp) [inline Setting-Up-A-React-ES6-Webpack-Project.md]: @alanbuchanan the previous was good. This change was not required.

**Pull request merged**